src/containers/Deposit/index.tsx

```diff
@@ -16,16 +16,15 @@ const Deposit = () => {
             <DepositListContainer>
                 {depositTokens.map((tokenData) => {
                     const tokenPoolDetails = poolDetails[tokenData.symbol]
-                    const userDepositInfo = tokenPoolDetails?.userInfo
-
                     return (
                         <DepositBlock
                             key={tokenData.symbol}
                             ticker={tokenData.symbol}
                             tvl={tokenPoolDetails?.tvl}
                             apr={tokenPoolDetails?.apy}
-                            userDeposit={userDepositInfo?.totalDepositAmount?.toString()}
-                            userRewards={userDepositInfo?.rewardDebtToken1?.toString()}
+                            // TODO: Update values once user pool deposit data is available
```

Comment:
As we discussed earlier you should be able to get these values from the SDK already. Feel free to use 0x8cc44a3Fe63E844f37CeE1C91f7b5bc4aD26639e as an address for testing